.github/workflows/doxygen.yml

```diff
@@ -39,6 +38,10 @@ jobs:
           if [ ! -f doxygen.err ]; then echo "No Doxygen error file found!"; exit 1; fi
           if [ -s doxygen.err ]; then echo "Doxygen failed with the following error(s):"; cat doxygen.err; exit 1; else echo "Doxygen passed."; fi
 
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
I guess for sake of experimenting, you could disable `if: ${{ github.event_name == 'push' || github.event_name == 'schedule' }}`.

Reply:
I had this disabled before, but it seems that actions making use of such `${{ secrets.ACTIONS_DEPLOY_KEY_DOXYGEN }}` a key may only be run on `push` or `schedule`.